docs/src/main/asciidoc/security-customization.adoc

```diff
@@ -150,6 +140,12 @@ public class RolesAugmentor implements SecurityIdentityAugmentor {
 }
 ----
 
+[NOTE]
+===
+If more than one custom `SecurityIdentityAugmentor` is registered then they will be considered equal candidates and called in random order.
+You can enforce the call order by implementing a default `SecurityIdentityAugmentor#priority` method.
```

Comment:
Higher priorities will be invoked first?

Reply:
@gastaldi Let me check :-)

Reply:
@gastaldi Hi George, how does it read now ?

Reply:
@sberyozkin excellent!